ccmlib/scylla_cluster.py

```diff
@@ -168,9 +174,9 @@ def stop_nodes(self, nodes=None, wait=True, gently=True, wait_other_notice=False
     def stop(self, wait=True, gently=True, wait_other_notice=False, other_nodes=None, wait_seconds=127):
         if self._scylla_manager:
             self._scylla_manager.stop(gently)
-        args = locals()
-        del args['self']
-        return self.stop_nodes(**args)
+        kwargs = dict(**locals())
```

Comment:
This is a change needed in CCM, regardless of our work,
without it, this part of code is failing under debugger